src/sql/src/session/vars.rs

```diff
@@ -1191,6 +1191,24 @@ pub static ENABLE_LAUNCHDARKLY: ServerVar<bool> = ServerVar {
     internal: true
 };
 
+/// Whether to use the new persist-txn tables implementation or the legacy
+/// one.
+pub static ENABLE_PERSIST_TXN_TABLES: ServerVar<bool> = ServerVar {
+    name: UncasedStr::new("enable_persist_txn_tables"),
+    value: &false,
+    description: "\
+        Whether to use the new persist-txn tables implementation or the legacy \
+        one.
+
+        Only takes effect on restart. Any changes will also cause clusterd \
+        processes to restart.
+
+        This value is also configurable via a Launch Darkly parameter of the \
+        same name, but we keep the flag to make testing easier. If specified, \
+        the flag takes precedence over the Launch Darkly param.",
+    internal: true,
+};
```

Comment:
We have the `feature_flags!` macro that help with defining features flags if you prefer to use that. You can see examples by grepping for `feature_flags!` in this file.

Reply:
Done